tools/BUILD

```diff
@@ -47,7 +47,7 @@ py_library(
 py_binary(
     name = "write_sbom",
     srcs = ["write_sbom.py"],
-    deps = [":sbom_lib"],
```

Comment:
The commit description should explain this commented out line. Can it be deleted instead of just commented out?

Reply:
That was an accident. Reverted.